### ISSUE TITLE
Crash continuously

### ISSUE THREAD

**Issue by KNemo27** [florisboard/florisboard#161]
App runs but keyboard crashes everytime after choosing as default keyboard. Unable to open keyboard. 
Also after clicking on Theme button in the app, it crashed multiple times.

**Environment information**
- FlorisBoard Version: 0.3.2
- Install Source: F-Droid
- Device: General Mobile GM9 Pro
- Android version, ROM: 9, Stock

**Steps to reproduce**
1. Go to Floris app, home page
2. Click on 'Florisboard is not selected as default input method, click here' -Yellow colored lines-
3. Switch keyboard
Select Floris as default input method
Trying to type
Keyboard crashes
4. Florisbord has stopped working continuously and default keyboard started

```
~~~ 1610307067356.stacktrace ~~~

com.squareup.moshi.JsonDataException: java.lang.IllegalArgumentException: No enum constant dev.patrickgold.florisboard.ime.text.key.KeyType.MODİFİER at $.arrangement[0][0].type
	at com.squareup.moshi.AdapterMethodsFactory$1.fromJson(AdapterMethodsFactory.java:94)
	at com.squareup.moshi.kotlin.reflect.KotlinJsonAdapter.fromJson(KotlinJsonAdapter.kt:84)
	at com.squareup.moshi.internal.NullSafeJsonAdapter.fromJson(NullSafeJsonAdapter.java:40)
	at com.squareup.moshi.CollectionJsonAdapter.fromJson(CollectionJsonAdapter.java:76)
	at com.squareup.moshi.CollectionJsonAdapter$2.fromJson(CollectionJsonAdapter.java:53)
	at com.squareup.moshi.internal.NullSafeJsonAdapter.fromJson(NullSafeJsonAdapter.java:40)
	at com.squareup.moshi.CollectionJsonAdapter.fromJson(CollectionJsonAdapter.java:76)
	at com.squareup.moshi.CollectionJsonAdapter$2.fromJson(CollectionJsonAdapter.java:53)
	at com.squareup.moshi.internal.NullSafeJsonAdapter.fromJson(NullSafeJsonAdapter.java:40)
	at com.squareup.moshi.kotlin.reflect.KotlinJsonAdapter.fromJson(KotlinJsonAdapter.kt:84)
	at com.squareup.moshi.internal.NullSafeJsonAdapter.fromJson(NullSafeJsonAdapter.java:40)
	at com.squareup.moshi.JsonAdapter.fromJson(JsonAdapter.java:43)
	at dev.patrickgold.florisboard.ime.text.layout.LayoutManager.loadLayout(LayoutManager.kt:63)
	at dev.patrickgold.florisboard.ime.text.layout.LayoutManager.loadLayout(LayoutManager.kt:45)
	at dev.patrickgold.florisboard.ime.text.layout.LayoutManager.mergeLayoutsAsync(LayoutManager.kt:128)
	at dev.patrickgold.florisboard.ime.text.layout.LayoutManager.computeLayoutFor(LayoutManager.kt:294)
	at dev.patrickgold.florisboard.ime.text.layout.LayoutManager$preloadComputedLayout$1.invokeSuspend(LayoutManager.kt:361)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
Caused by: java.lang.IllegalArgumentException: No enum constant dev.patrickgold.florisboard.ime.text.key.KeyType.MODİFİER
	at java.lang.Enum.valueOf(Enum.java:258)
	at dev.patrickgold.florisboard.ime.text.key.KeyType.valueOf(Unknown Source:2)
	at dev.patrickgold.florisboard.ime.text.key.KeyType$Companion.fromString(KeyType.kt:40)
	at dev.patrickgold.florisboard.ime.text.key.KeyTypeAdapter.fromJson(KeyType.kt:48)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.squareup.moshi.AdapterMethodsFactory$AdapterMethod.invoke(AdapterMethodsFactory.java:335)
	at com.squareup.moshi.AdapterMethodsFactory$5.fromJson(AdapterMethodsFactory.java:257)
	at com.squareup.moshi.AdapterMethodsFactory$1.fromJson(AdapterMethodsFactory.java:90)
	... 22 more

```

**Comment by patrickgold** [comment]
I am aware of this issue and it should normally be fixed with v0.3.3. Related to #135. Could you give me feedback about this after you've received v0.3.3 if it has been fixed (will be released sometime next week)? Thanks in advance!

**Comment by patrickgold** [comment]
@KNemo27 Has this crash been resolved for you in the newest version v0.3.4/v0.3.5?

**Comment by patrickgold** [comment]
Same bug as #135 , has been fixed and tested on an emulated Turkish Android 10 device. Closing this issue now.
